app/src/main/java/com/benefit/activities/SignInActivity.java

```diff
@@ -50,7 +51,9 @@
  */
 public class SignInActivity extends AppCompatActivity implements OnMapReadyCallback, OnMarkerDragListener {
 
-    public enum LoginState{NOT_SIGN_IN, LOGGING_IN, SIGN_IN_GET_USER, NEW_USER_SIGN_UP, FINISH};
+    public enum LoginState {NOT_SIGN_IN, LOGGING_IN, SIGN_IN_GET_USER, NEW_USER_SIGN_UP, FINISH}
```

Comment:
we have an enum package under model package. you can add this to model.enum.signIn or something like that, to maintain the single mapping principle.

Reply:
fixed